xdrip-get-entries.sh

```diff
@@ -502,6 +501,11 @@ function check_sensor_stop()
           echo "stopJSON = $stopJSON"
           # below done so that next time the egrep returns positive for this specific message and the log reads right
           echo "Already Processed Sensor Stop Message from Nightscout at $createdAt" >> ${LDIR}/nightscout-treatments.log
+          # Always clear LSR cache for any g6 start / stop
+          if [ "$txType" == "g6" ]; then
```

Comment:
i'd prefer to be able to turn this off by a xdripjs.json variable, so older g6 tx's can keep their calibration.

Reply:
Good point. I was able to use the version firmware to only do these clears for a g6 with new firmware version.